renderdoc/driver/gl/gl_counters.cpp

```diff
@@ -382,7 +381,7 @@ vector<CounterResult> GLReplay::FetchCounters(const vector<GPUCounter> &allCount
 
   vector<GPUCounter> counters;
   std::copy_if(allCounters.begin(), allCounters.end(), std::back_inserter(counters),
-               [](const GPUCounter &c) { return !IsAMDCounter(c); });
+               [](const GPUCounter &c) { return IsGenericCounter(c); });
```

Comment:
This is a good change, but can you please extend this to call the function in the corresponding places for the other APIs please? They've just added more negative conditions which has gotten a bit silly at this point.